CMakeLists.txt

```diff
@@ -199,6 +199,19 @@ if (OPM_ENABLE_PYTHON)
     else()
       find_package(Python3 REQUIRED COMPONENTS Interpreter)
     endif()
+    # Make sure we fail gracefully here without setuptool
+    execute_process(COMMAND ${Python3_EXECUTABLE} -c "import setuptools"
+      WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+      ERROR_VARIABLE SETUPTOOL_ERROR OUTPUT_VARIABLE SETUPTOOL_OUT
+      RESULT_VARIABLE SETUPTOOL_RESULT)
+    message("ERROR_VARIABLE=${SETUPTOOL_ERROR} OUTPUT_VARIABLE=${SETUPTOOL_OUT} RESULT_VARIABLE=${SETUPTOOL_RESULT}")
```

Comment:
Is this leftover debugging or an intentional, user-directed message?

Reply:
Outch leftover debugging. fixed.